.travis.yml

```diff
@@ -24,9 +25,19 @@ gemfile:
 - Gemfile.activerecord40
 - Gemfile.activerecord41
 - Gemfile.activerecord42
+- Gemfile.activerecord50
 
 matrix:
   allow_failures:
-  - rvm: ruby-head
-  - rvm: jruby-head
-  - rvm: jruby-19mode
+    - rvm: ruby-head
+    - rvm: jruby-head
+    - rvm: jruby-19mode
+  exclude:
+    - rvm: "1.9"
+      gemfile: Gemfile.activerecord50
```

Comment:
I think we should drop support for Ruby 1.9 at this point, and maybe also 2.0, because they're both EOL.